src/patternfly/components/Dropdown/dropdown.scss

```diff
@@ -416,6 +433,16 @@
   }
 }
 
+.pf-c-dropdown__menu-item-description {
+  font-size: var(--pf-c-dropdown__menu-item-description--FontSize);
+  color: var(--pf-c-dropdown__menu-item-description--Color);
+}
+
+.pf-c-dropdown__menu-item-main {
+  display: flex;
+  align-items: center;
+}
```

Comment:
looks like this is still here and doesn't need to be since you added it when it's a child of `.pf-c-dropdown__menu-item.pf-m-icon`